src/bp/ui-shared/src/BaseDialog/index.tsx

```diff
@@ -5,7 +5,7 @@ import React, { FC } from 'react'
 import { BaseDialogProps } from './typings'
 
 export const BaseDialog: FC<BaseDialogProps> = props => {
-  let width = 500
+  let width = 360
```

Comment:
I'm worried this might break a lot of existing dialogs, could you add a 'sm' size condition below instead for 360?